test/aws/ssh_bastion.yml

```diff
@@ -1,51 +1,205 @@
 ---
-- name: Create ssh bastion project
-  oc_project:
-    name: openshift-ssh-bastion
-    description: openshift-ssh-bastion
-    node_selector: ""
+- name: Create ssh bastion namespace
+  k8s:
     kubeconfig: "{{ kubeconfig_path }}"
-
-- name: Create ssh keys secret
-  oc_secret:
+    kind: Namespace
+    name: "{{ ssh_bastion_namespace }}"
     state: present
-    namespace: openshift-ssh-bastion
-    name: ssh-host-keys
-    kubeconfig: "{{ kubeconfig_path }}"
-    files:
-    - name: ssh_host_rsa_key
-      path: ../../inventory/dynamic/injected/ssh-privatekey
-    - name: sshd_config
-      path: files/sshd_config
+
+- name: Create ssh bastion keys secret
+  k8s:
+    kubeconfig: "{{ kubeconfig_path }}"
+    resource_definition:
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: ssh-host-keys
+        namespace: "{{ ssh_bastion_namespace }}"
+      data:
+        ssh_host_rsa_key: "{{ lookup('file', '../../inventory/dynamic/injected/ssh-privatekey') | b64encode }}"
+        sshd_config: "{{ lookup('file', 'files/sshd_config') | b64encode }}"
   no_log: true
 
-- name: Create ssh bastion
-  command: "oc --kubeconfig={{ kubeconfig_path }} apply -f files/"
+- name: Create ssh bastion service
+  k8s:
+    kubeconfig: "{{ kubeconfig_path }}"
+    resource_definition:
```

Comment:
I'd prefer `definition: "{{ lookup('template', 'files/01_service.yml') }}"` here instead to make the playbook more readable

Reply:
Ok, I'll rework this.